sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/Internals/SchemaConstants.cs

```diff
@@ -115,5 +115,7 @@ internal static class SchemaConstants
         public const int TelemetryEnvelope_Name_MaxLength = 1024;
         public const int TelemetryEnvelope_Time_MaxLength = 64;
         public const int TelemetryEnvelope_InstrumentationKey_MaxLength = 40;
+
+        public const string DefaultSchemaVersion = "1.21.0";
```

Comment:
This file was intended for Azure Monitor Schema Constants (see summary at top).
I would recommend adding a separate file for OTel Constants.

Reply:
This is a bogus value and may not fit anywhere. There are no validations based on this schema version. I found its a best place to reside until we get the older version removed. Creating a new constant may not be a good idea for bogus value. 

On a side note this class should have not been named as SchemaConstants.

Reply:
> This is a bogus value and may not fit anywhere.

Can you add a comment stating as much and that this would be removed in the future?

> On a side note this class should have not been named as SchemaConstants.

Agreed.

Reply:
We plan to remove this variable itself in follow up PRs.